README.md

```diff
@@ -266,6 +264,10 @@ ElasticsearchEvolution.configure()
 ```
 
 ## 6 changelog
+### v0.3.0-SNAPSHOT
+- version upgrade elasticsearch-rest-high-level-client to 7.5.0 (Es version < 7.5.0 are no longer supported)
+- version upgrade testcontainer to 1.15.0 
```

Comment:
version update mainly dto run the tests on macOs https://github.com/testcontainers/testcontainers-java/pull/3159